homeassistant/components/modbus/switch.py

```diff
@@ -198,9 +204,7 @@ def __init__(
         state_off,
     ):
         """Initialize the register switch."""
-        self._hub = hub
-        self._name = name
-        self._slave = slave
+        super().__init__(hub, name, slave)
```

Comment:
You disable pylint, and then call super() ??

Reply:
Preference is not to disable lint.

Reply:
I didn't disable Pylint by purpose. I will take care of this.

Reply:
I didn't disable pylint. It was disabled in original code for `switch.py`. I added the call to `super()` and removed that pylint comment.